lunch/readme.md

```diff
@@ -1,21 +1,34 @@
-# Replace with recipe name
+# Turkey sandwich
 
-## Servings: 
+## Servings: 4
 
-## Time to prepare: 
+## Time to prepare: 20 minutes
 
 ## Ingredients:
-
+Turkey <br/>
+Bread <br/>
+Lettuce <br/>
+Mayo <br/>
+Tomato
 
 ## Equipment needed:
+Hands <br/>
+Knife
 
 
 ## Steps for preparation:
-
-
+step 1: take two slices of bread <br/>
+step 2: take your knife and spread mayo on the face of one piece of bread <br/>
+step 3: slice tomato <br/>
+step 4: put sliced tomato, turkey, and lettuce on the bread <br/>
```

Comment:
Adding a sliced tomato was a great choice

Reply:
Agreed, really helps for sandwiches